submissions/AnnaStoyano/js-dom/script.js

```diff
@@ -0,0 +1,105 @@
+const downloadInfoFromJson = function () {
+    let promise = fetch('./information.json');
+    promise.then(res => res.json())
+        .then(data => {
+            addMenuItems(data.faculties);
+            addContentItems(data.faculties);
+            return data;
+        })
+        .then(data=>{
+            menuHandler(data);
```

Comment:
Why you used twice the method `then`?
Think about constructive code
This function only returns data. You can write data to a variable and call a function. Calling all four functions outside of this function
Also, think about network conflicts and use the catch method to inform users of the problem.

Reply:
Do you mean move this functions to one .then()? I thought that fetch is async, so before we call event handler we should wait until all data have downloaded, that's why I decided to use 2 different .then().

Reply:
The method `then` is already working with the loaded data. You can read about [async/wait](https://developer.mozilla.org/ru/docs/Learn/JavaScript/Asynchronous/Async_await) and use this in final projects.